classes/Cart.php

```diff
@@ -3790,7 +3790,7 @@ public function checkDiscountValidity($obj, $discounts, $order_total, $products,
      *
      * @return array Cart details
      */
-    public function getSummaryDetails($id_lang = null, $refresh = false)
+    public function getSummaryDetails($id_lang = null, $refresh = false, $splitGift = true)
```

Comment:
```suggestion
    public function getSummaryDetails($id_lang = null, $refresh = false, bool $splitGift = true)
```

Reply:
since the other properties are not typed. I didn't want to typehint this one. but why not ?

Reply:
In all cases, it's a BC :)